.gitignore

```diff
@@ -26,6 +26,7 @@ tmtags
 
 ## Project (general)
 tags
+tmp
```

Comment:
I was in doubt whether this entry belongs in the project's or in the global gitignore... :man_shrugging:

Reply:
Sorry, a little confused. You mean in another `gitignore` file completely? Oh wait you mean in just the same file but a different section? I think it's fine as is. 👍🏻

Reply:
Normally I put on my global gitignore (`~/.git/ignore`) stuff specific to my workflow but unrelated to the project (for example, that temp file my editor "x" generates) , but in the project's `.gitignore` stuff specific to the project. In this case I was not sure in which category "tmp" falls.

Anyways, seems fine as is indeed :+1:!

Reply:
Ah ok that’s what I thought you meant but had me confused haha. Yes I think it’s fine here. Doesn’t hurt. Thanks! 👍🏻